src/app/static/src/app/components/plots/bar/Barchart.vue

```diff
@@ -50,7 +50,7 @@ export default defineComponent({
         });
 
         const chartDataGetter = store.getters["plotSelections/barchartData"];
-        const chartData = ref<BarChartData>({datasets:[], labels: [], maxValuePlusError: 0});
+        const chartData = ref<BarChartData<(number | null)[]>>({datasets:[], labels: [], maxValuePlusError: 0});
```

Comment:
this is the type of `data` in the `datasets` array (`datasets` is a key of `ChartData` more on this later)